workers/run_tests.sh

```diff
@@ -11,11 +11,19 @@ cd $script_directory
 # move up a level
 cd ..
 
+# Set up the test data volume directory if it does not already exist
+volume_directory="$script_directory/test_volume"
+if [ ! -d "$volume_directory" ]; then
+    mkdir $volume_directory
```

Comment:
Do you need to consider the case of `$script_directory/test_volume` exists but is not a directory? In that case `mkdir` command will fail.